RNTester/Podfile

```diff
@@ -125,6 +125,13 @@ post_install do |installer|
         puts '  adding arm64e to ' + config.name
       end
     end
+    # TODO(macOS ISS#2323203): the internal Microsoft build pipeline needs macOS arm64 slices
+    if target.platform_name == :osx
+      target.build_configurations.each do |config|
+        (config.build_settings['ARCHS'] ||= ['$(ARCHS_STANDARD)']) << 'arm64'
```

Comment:
This may not be necessary. In standard Xcode 12, ARCHS_STANDARD includes arm64. This definitely won't hurt anything though.

Reply:
I based it off what we did for our CI loops in FluentUI-Apple where we explicitly set the ARCHS to arm64